File: src/main/java/com/intellij/tapestry/intellij/view/nodes/LibraryNode.java
package com.intellij.tapestry.intellij.view.nodes;

import com.intellij.icons.AllIcons;
import com.intellij.ide.projectView.PresentationData;
import com.intellij.openapi.module.Module;
import com.intellij.psi.PsiDirectory;
import com.intellij.tapestry.core.model.TapestryLibrary;

public class LibraryNode extends PackageNode {

    public LibraryNode(TapestryLibrary library, PsiDirectory psiDirectory, Module module) {
        super(library, psiDirectory, module);

        init(psiDirectory, new PresentationData(psiDirectory.getName(), psiDirectory.getName(),
                                                AllIcons.Nodes.PpLib, null));
    }
}

File: src/main/kotlin/com/intellij/tapestry/intellij/view/nodes/LibraryNode.kt
package com.intellij.tapestry.intellij.view.nodes

import com.intellij.icons.AllIcons
import com.intellij.ide.projectView.PresentationData
import com.intellij.openapi.module.Module
import com.intellij.psi.PsiDirectory
import com.intellij.tapestry.core.model.TapestryLibrary

class LibraryNode(library: TapestryLibrary, psiDirectory: PsiDirectory, module: Module) :
    PackageNode(library, psiDirectory, module) {
    init {
        init(psiDirectory, PresentationData(psiDirectory.name, psiDirectory.name, AllIcons.Nodes.PpLib, null))
    }
}
